syft/pkg/type.go

```diff
@@ -20,6 +20,7 @@ const (
 	GoModulePkg      Type = "go-module"
 	RustPkg          Type = "rust-crate"
 	KbPkg            Type = "msrc-kb"
+	PubPkg           Type = "pub"
```

Comment:
Ahh, so this is possibly what is going to be included in the packageurl spec: https://github.com/package-url/purl-spec/blob/master/PURL-TYPES.rst#other-candidate-types-to-define

Reply:
Yes I have a PR https://github.com/package-url/purl-spec/pull/156